app/controllers/content.js

```diff
@@ -22,6 +22,12 @@ export default Ember.Controller.extend({
              this.set('authors', contributors));
     }),
 
+    applyMathJax: Ember.observer('model', function() {
```

Comment:
Can you please clarify why this observer tracks the model (aka blank preprint)? 

The `model` hook is a blank preprint (to hold subject and DOI, mainly). But since most fields on the page with mathematical data (like subject and title) live on the `node`, it seems strange for this observer to watch the preprint.

If you just want this to run once when the page starts, you might consider moving the logic to `setupController `instead. Otherwise, it's not clear to me this code is firing when you expect, for the reasons expected.